src/dbup-postgresql/PostgresqlExtensions.cs

```diff
@@ -150,7 +150,7 @@ private static void PostgresqlDatabase(this SupportedDatabasesForEnsureDatabase
         var logMasterConnectionStringBuilder = new NpgsqlConnectionStringBuilder(masterConnectionStringBuilder.ConnectionString);
         if (!string.IsNullOrEmpty(logMasterConnectionStringBuilder.Password))
         {
-            logMasterConnectionStringBuilder.Password = string.Empty.PadRight(masterConnectionStringBuilder.Password.Length, '*');
+            logMasterConnectionStringBuilder.Password = "******";
```

Comment:
```suggestion
            logMasterConnectionStringBuilder.Password = "******"
```
Small typo on these three lines, which is why the build failed. I'll add suggestions to the others as well, so you can quickly fix them.